test/browser_tests/page_objects/page_about-us.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+function AboutUs() {
+  this.get = function() {
+    browser.get( '/about-us/' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle(); };
+  this.sidebar = element.all( by.css( 'aside.activity' ) );
```

Comment:
Agreed. `activityBlock` is more descriptive. I'll make the change in the next round of edits.